### ISSUE TITLE
Long building names are cut off in several views (instead of being wrapped)

### ISSUE THREAD

**Issue by olifre** [indico/indico#6467]
**Describe the bug**
When using addresses as building names (as is common for a distributed Campus without "building numbers"), building names easily do get lengthy and may be cut off in various views, even though there would be enough space to wrap it or use more horizontal space:
![grafik](https://github.com/user-attachments/assets/f10613b0-9375-45c8-98cd-17b67b2a4dfd)
The same also happens in the "List of Rooms" view. 

**To Reproduce**
1. Use `{building}/{number} ({site})` as room name format. 
2. Set a street name and number for the building name, e.g. "Käthe-Kümmel-Straße 1". 
3. Observe this is too long and cut off in various views. 

**Expected behavior**
Wrapping of lengthy building / room names. 

**Screenshots**
see above :wink:

**Additional context**
I considered putting "site" or another building identifier in front, but sadly, there is no common short building name on our Campus apart from the actual address (e.g. several institutes in one building, no "name" for the actual building itself). Hence, using the street name / address is the most unique denominator., which also seems universally unique for distributed environments. 

PS: If you think this is rather a feature request, feel free to re-tag accordingly.

**Comment by ThiefMaster** [comment]
Not sure wrapping them would look good, I think ellipsizing them (ideally w/ a tooltip showing the full name on hover) would be better...

FWIW the "building" field was pretty much meant for simple building numbers ("513", "A38", etc.), not long street names, but I guess some sites (like yours) don't have "simple" building numbers that are unique w/o adding the street name?

**Comment by olifre** [comment]
> FWIW the "building" field was pretty much meant for simple building numbers ("513", "A38", etc.), not long street names, but I guess some sites (like yours) don't have "simple" building numbers that are unique w/o adding the street name?

Indeed, exactly that is the case — we don't have a site-wide numbering of buildings, or a unique naming of all buildings, so street names is the only way to go for. 
Maybe something like putting the room number in the next row would be possible, i.e. ellipsize long building names, but never cut off the room number?

**Comment by olifre** [comment]
> A tooltip imo would not be that helpful here if you're looking at multiple rooms and have to remember which is which.

Fully agreed. Note that there is a similar issue on the "List of Rooms" page:
![grafik](https://github.com/user-attachments/assets/16ea0f29-f724-4c6f-8e36-73a563e6dc01)

This also applies to some "rooms" at CERN Indico such as the BBQ areas, for which the building number is visible just fine, but the "readable" name of the BBQ area (Prevessing etc.) is cut off.

**Comment by tomasr8** [comment]
> Note that there is a similar issue on the "List of Rooms" page:

We could try a flexible height there as well - especially when there's no room description there's quite a bit of unused space that can be taken up by the room name.
